src/apis/doctor/doctor-store.dao.js

```diff
@@ -63,8 +63,25 @@ class DoctorStoreDao {
     }
 
     /**
-     * Delete method
+     * Get by user id method
      */
+    getByUserId(doctorStore, userId, type, callback) {
+            return sequelize.transaction(function(t) {
+                return doctorStoreModel.doctor_store
+                    .readAll(doctorStore, {
+                        where: {
```

Comment:
No need to pass the entire object when you are specifically reading using ID

Reply:
yeah we need not pass the full object but we are not passing here the object is dependant of the call from the service that will include the json mapped value.